#!/bin/sh /usr/share/dpatch/dpatch-run
## 03-migrate.dpatch by Brian DeRocher <brian@derocher.org>
##
## DP: Correct SQL syntax in migrate script (Closes: #467517).

@DPATCH@

diff -Naurp openerp-server.orig/doc/migrate/3.4.0-4.0.0/pre.py openerp-server/doc/migrate/3.4.0-4.0.0/pre.py
--- openerp-server.orig/doc/migrate/3.4.0-4.0.0/pre.py	2008-11-03 21:33:56.000000000 +0000
+++ openerp-server/doc/migrate/3.4.0-4.0.0/pre.py	2008-11-09 09:09:49.000000000 +0000
@@ -123,7 +123,7 @@ cr.commit()
 
 for line in (
         "ALTER TABLE ir_module_module ADD demo BOOLEAN",
-        "ALTER TABLE ir_module_module SET demo DEFAULT False",
+        "ALTER TABLE ir_module_module alter column demo set DEFAULT False",
         "DELETE FROM ir_values WHERE VALUE LIKE '%,False'",
         """UPDATE ir_ui_view set arch='<?xml version="1.0"?><tree string="Menu" toolbar="1"><field icon="icon" name="name"/></tree>' where name='ir.ui.menu.tree' and type='tree' and field_parent='child_id'""",
     ):
